### ISSUE TITLE
Minecraft music still plays when playing jukebox

### ISSUE THREAD

**Issue by DanielMC101** [Wilyicaro/Legacy-Minecraft#103]
Also, another thing is that creative music barely plays when on creative mode.

https://github.com/Wilyicaro/Legacy-Minecraft/assets/39068731/f625a581-a20e-40ef-871a-4a11b5c2e165

**Comment by Quartzmaven** [comment]
This appears to be not related to the mod itself. I installed "Infinite Music" in my own modpack, all it does is to remove delay and fix problems with the jukebox. Yes, these can be set in mod settings.
https://modrinth.com/mod/infinite-music

https://github.com/Wilyicaro/Legacy-Minecraft/assets/81502335/4a1b324f-60a4-442b-b435-f8c2dacc10e5

**Comment by Chilly-roll** [comment]
why wouldn't it be related to the mod? it's supposed to include console features, right? that's a console feature..

as for the creative music thing, java edition includes the "music.game" event in "music.creative", which leads to survival music playing in creative as well. you'd have to remove that in sounds.json for creative to solely be creative music

**Comment by Wilyicaro** [comment]
> why wouldn't it be related to the mod? it's supposed to include console features, right? that's a console feature..
> 
> as for the creative music thing, java edition includes the "music.game" event in the "music.creative" event, which leads to survival music playing in creative as well. you'd have to remove that in sounds.json for creative to solely be creative music

Yeah, but the issue title suggests it's a mod problem, not just a LE feature not added yet
